MIT.md

```diff
@@ -1,4 +1,5 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2023, [Mohamed Gamil Eldimardash]
+## Copyright 2023, [Lucas Comamala Garcia]
 
 ###### Please delete this line and the next one
```

Comment:
- [ ] Your license is currently a placeholder, kindly edit ```[APP TYPE]``` with the name of your app and remove the commented line that it says you should delete. This would aid in making your documentation more professional